### PR TITLE
adds examples to README and refactors commands for Language samples

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: language/api/README.md [hunk 1]
```diff
@@ -8,6 +8,156 @@ Library for PHP][google-cloud-php] to make REST calls as well as
...
+## The client library
+
+This sample uses the [Google Cloud Client Library for PHP][google-cloud-php].
+You can read the documentation for more details on API usage and use GitHub
+to [browse the source][google-cloud-php-source] and  [report issues][google-cloud-php-issues].
+
+[google-cloud-php]: https://googlecloudplatform.github.io/google-cloud-php
+[google-cloud-php-source]: https://github.com/GoogleCloudPlatform/google-cloud-php
+[google-cloud-php-issues]: https://github.com/GoogleCloudPlatform/google-cloud-php/issues
+[google-cloud-sdk]: https://cloud.google.com/sdk/
```
Comment (sgreenberg) [inline language/api/README.md]: Maybe at the bottom of this file add this same Troubleshooting text regarding the date issue? 

https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/speech/api

File: language/api/README.md [hunk 2]
```diff
@@ -8,6 +8,156 @@ Library for PHP][google-cloud-php] to make REST calls as well as
...
+
+    Available commands:
+      all        Analyze syntax, sentiment and entities in text.
+      entities   Analyze entities in text.
+      help       Displays help for a command
+      list       Lists commands
+      sentiment  Analyze sentiment in text.
+      syntax     Analyze syntax in text.
+
+### Run Analyse Entities
```
Comment (sgreenberg) [inline language/api/README.md]: You spell Analyse with an s in the UK, but with a z in the US. Probably should switch to a z. This change is throughout the README file.

File: language/api/README.md [hunk 3]
```diff
@@ -8,6 +8,156 @@ Library for PHP][google-cloud-php] to make REST calls as well as
...
+                        content: 'San Jose'
+                        beginOffset: 18
+                    type: PROPER
+    language: en
+
+### Run Analyse Sentiment
+
+To run the Analyse Sentiment sample:
+
+    $ php language.php sentiment 'I know the way to San Jose'
```
Comment (sgreenberg) [inline language/api/README.md]: As sentence-level sentiment is a key thing to show off, can you include two sentences?

File: language/api/README.md [hunk 4]
```diff
@@ -8,6 +8,156 @@ Library for PHP][google-cloud-php] to make REST calls as well as
...
+                beginOffset: 0
+            sentiment:
+                magnitude: 0.3
+                score: 0.3
+
+### Run Analyse Syntax
+
+To run the Analyse Syntax sample:
+
+    $ php language.php sentiment 'I know the way to San Jose'
```
Comment (sgreenberg) [inline language/api/README.md]: Should be a call to analyze syntax not sentiment.

File: spanner/test/spannerTest.php [hunk 5]
```diff
@@ -33,10 +33,6 @@ class spannerTest extends \PHPUnit_Framework_TestCase
     /* @var $instance Instance */
```
Comment (sgreenberg) [inline spanner/test/spannerTest.php]: Was this spannerTest file included in the PR by mistake?

Comment (bshaffer) [inline spanner/test/spannerTest.php]: No, I refactored the tests :)

**Pull request merged**